lib/tools/apk-utils.js

```diff
@@ -544,6 +550,7 @@ apkUtilsMethods.setDeviceLanguageCountry = async function (language, country) {
   }
 
   if (wasSettingChanged) {
+    log.info("To apply locale setting via setting persist.sys.locale command, reboot the target device is required.");
```

Comment:
-> Rebooting the device in order to apply new locale via 'setting persist.sys.locale' command...